.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+data/*
+study/*
```

Comment:
maybe these should be excluding `*.sh` since we do prolly want to add init script changes? I forget the ignore syntax for this

Reply:
if *.sh was excluded then we would exclude init script changes..

data/ and study/ seem to just be downloaded using init.sh. So im confused by your comment

Reply:
I meant it would prolly make sense to not ignore changes to https://github.com/cBioPortal/cbioportal-docker-compose/blob/master/data/init.sh  and https://github.com/cBioPortal/cbioportal-docker-compose/blob/master/study/init.sh

Reply:
oh you mean exclude *sh from the ignore. got it